CTP/README.md

```diff
@@ -36,18 +33,38 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
 	URL:  http://127.0.0.1:8888    
     ```
 	
-* Please open the URL with your browser.
+* Please open the URL with your browser
   
 ## How To Execute
-- **SQL** 
+- **SQL**
  - Prepare
-	* Checkout test cases from our GitHub projects or make your own test cases.
-	* Install CUBRID and make sure your environment variable of ``CUBRID`` is correctly set.
-	* Check configuration files, for **SQL/Medium** test, you can modify parameters of ``conf/sql.conf`` or ``conf/medium.conf``. 
-	* Set ``scenario`` of ``[sql]`` section to the test cases directory.
-	* Set ``data_file`` of ``[sql]`` section to the directory path of initial data files for **Medium** test.
-	* ``test_mode=yes`` and ``java_stored_procedure=yes`` parameters must be set for **SQL/Medium** test.
-	* Please see ``conf/sql.conf`` or  ``conf/medium.conf`` for details about other parameters.
+ 	* Install CUBRID and make sure your environment variable of ``CUBRID`` is set correctly
+ 	* Check out scenarios from CUBRID GitHub projects or prepare your own test cases for testing

```

Comment:
-> * Check out scenarios from [cubrid-testcases](https://github.com/CUBRID/cubrid-testcases) project or prepare your own test cases for testing